.github/workflows/pull_request_docker.yml

```diff
@@ -1,15 +1,14 @@
-name: Pull Request (Docker)
+name: Docker PR
 
 on:
   pull_request:
     paths:
-    - '/Cargo.lock'
-    - '/Dockerfile'
+    - 'Cargo.lock'
+    - 'Dockerfile'
```

Comment:
confirming this workflow is specifically for checking whether the docker build is working? is it possible that a change could cause the `Rust PR` workflow to pass but this workflow to fail? any reason not to run this all the time?

Reply:
yes, it's just checking that the dockerfile hasn't regressed. This can't be impacted by code changes, only by dependency/structure changes.

Reply:
also, this isn't even a required check currently, just informational, since that dockerfile isn't in the critical path for anything. starting with just the essentials in the critical path until i have a better handle on it all